projects/storefrontapp-e2e-cypress/cypress/helpers/payment-methods.ts

```diff
@@ -199,23 +196,25 @@ export function paymentMethodsTest() {
   });
 }
 
-function getWaitAliases() {
-  const shippingPage = waitForPage(
-    '/checkout/shipping-address',
-    'getShippingPage'
-  );
-
-  const deliveryPage = waitForPage(
-    '/checkout/delivery-mode',
-    'getDeliveryPage'
-  );
-
-  const paymentPage = waitForPage(
-    '/checkout/payment-details',
-    'getPaymentPage'
-  );
-
-  const reviewPage = waitForPage('/checkout/review-order', 'getReviewPage');
-
-  return { shippingPage, deliveryPage, paymentPage, reviewPage };
+function addPaymentMethod(paymentDetail: PaymentDetail) {
+  cy.get('.cx-total')
```

Comment:
Why do you add the payment with a request and not using the flow? I guess it's not a problem since the test is for my-account but I want to know why.

Reply:
exactly what you said, the test is for my account and not testing exactly the flow of checkout

Previously, when we were still ducklings for cypress, we did all tests manually, such as navigating to the spot that we want to do. In this case, we were doing checkout to add a payment method in the checkout step since we had no other ways to do this. 

Adding a payment in checkout is being covered in important steps like checkout-flow